src/storage/mod.rs

```diff
@@ -49,7 +49,7 @@ impl Default for StorageTokenInfo {
 /// Trait for discovering available storage database backend types.
 ///
 /// Each backend implementation provides a static instance of this trait
-/// (e.g., `json::DBINFO`, `sqlite::DBINFO`) which is registered in the
+/// (e.g., `sqlite::DBINFO`) which is registered in the
```

Comment:
should we add the nssdb one here instead?

Reply:
I think one example is enough